sources/packages/backend/apps/api/src/esdc-integration/e-cert-integration/e-cert-full-time-integration/e-cert-files/e-cert-file-footer.ts

```diff
@@ -1,8 +1,5 @@
-import { StringBuilder } from "../../../../utilities/string-builder";
-import {
-  SPACE_FILLER,
-  NUMBER_FILLER,
-} from "../../../models/esdc-integration.model";
+import { StringBuilder } from "@sims/integration/utilities";
+import { NUMBER_FILLER, SPACE_FILLER } from "@sims/integration";
```

Comment:
Same comment gave before for date formats, these ones also are supposed to belong to the integration scope.